python-package/lightgbm/callback.py

```diff
@@ -215,6 +221,10 @@ def _callback(env):
         if not enabled[0]:
             return
         for i in range_(len(env.evaluation_result_list)):
+            metric_key = env.evaluation_result_list[i][1]
```

Comment:
I think `metric_key` variable is not needed anymore. It's enough to check `env.evaluation_result_list[i][1].split(" ")[0] == "train"` directly.